aziotctl/src/system.rs

```diff
@@ -38,12 +41,24 @@ pub struct LogLevelOptions {
     log_level: log::Level,
 }
 
-pub fn system(options: Options) -> Result<()> {
+#[derive(StructOpt)]
+#[structopt(about = "Reprovision device with IoT Hub")]
+pub struct ReprovisionOptions {
+    #[structopt(
+        value_name = "Identity Service URI",
+        long,
+        default_value = "unix:///var/run/aziot/identityd.sock"
```

Comment:
If you're going to keep it as a parameter, remove the `/var` from this to be consistent. All the other endpoints use `/run/aziot/...

Reply:
Fixed